ansible/roles/controller/tasks/deploy.yml

```diff
@@ -14,7 +14,6 @@
     path: "{{ whisk_logs_dir }}/controller{{ groups['controllers'].index(inventory_hostname) }}"
     state: directory
     mode: 0777
-  become: true
```

Comment:
in general there are many of these there are a problem - addressed in #2536 since it seems wrong to begin with.